psi4/src/export_mints.cc

```diff
@@ -1544,7 +1546,9 @@ void export_mints(py::module& m) {
                                                         "Extracts information from a wavefunction object, \
                                                                           and writes it to an FCHK file")
         .def(py::init<std::shared_ptr<Wavefunction>>())
-        .def("write", &FCHKWriter::write, "Write wavefunction information to file", "filename"_a);
+        .def("write", &FCHKWriter::write, "Write wavefunction information to file", "filename"_a)
+        .def("SCF_Dtot",&FCHKWriter::SCF_Dtot,py::return_value_policy::reference_internal)
+        .def("set_postscf_density_label", &FCHKWriter::set_pHF_density_label, "Set base label for post-SCF density, e.g. ' CC Density'.", "label"_a);
```

Comment:
I'm not sure what's happening with the semicolons here.

Reply:
The semicolon is needed at the end of the class.